README.md

```diff
@@ -34,17 +39,46 @@ class TextController extends Controller
 {
     public function index()
     {
+        //sending sms
         $message = "Hello Word"; //String
         
-        $receipients = [255717599994, '+255 717 599-994', '0717  599  994']; 
-        //recipients is an array of receiver nuumber in integers like 255717599994, and 717599994 
+        $recipients = [255717599994, '+255 717 599-994', '0717  599  994']; 
+        //recipients is an array of receiver phone number in integers like 255717599994, and 717599994 
         //or strings like '255717599994', '0717599994', '717599999', '+255-717-599-994', and '0717 599 994',
         
-        return LaravelSms::send($message, $recipients);
+        //create a new instance of the class LaravemSms class and pass the message and recipients.
+        $sms = new LaravelSms();
+        $sms->send($message, $recipients);
+    
+        return $sms;
     }
 }
 ```
+### Checking SMS Balance
+
+In your controller use like this
+
+```php
+namespace App\Http\Controllers;
 
+use Keenops\LaravelSms\LaravelSms;
+
+class TextController extends Controller
+{
+    public function index()
+    {        
+        //create a new instance of the class LaravemSms class and call the method viewBalance on that instance.
+        $sms = new LaravelSms();
```

Comment:
@nickyraph README example should emphasize on using Facade  instead of direct class initialization 

more info https://laravel.com/docs/8.x/facades